### ISSUE TITLE
Add support for today's date in title format fields

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/all-in-one-seo-pack#606]
Requested here - https://wordpress.org/support/topic/add-todays-date/

Add todays date as an option for the title format fields.

**Comment by michaeltorbert** [comment]
This should work for posts, pages, and post types in the doc title.

**Comment by wpsmort** [comment]
@michaeltorbert I have confirmed that this works for pages, posts, attachment pages and CPTs for doc title and description formats.

I did find one issue though.  The date appears in the Preview Snippet title only for the homepage, it does not appear in the Preview Snippet title for any other page or post or CPT.  See screenshots:

<img width="701" alt="screen-capture-1" src="https://cloud.githubusercontent.com/assets/17548525/19745340/3514ccf8-9b9f-11e6-90b4-5893087a3322.png">

<img width="1113" alt="screen-capture" src="https://cloud.githubusercontent.com/assets/17548525/19745407/7cb08552-9b9f-11e6-9c77-06db1e9890b7.png">
